### DISCUSSION TITLE
群晖运行docker问题

### DISCUSSION THREAD

**Discussion by waiguoren001** [go-olive/olive#103] [Q&A]
群晖docker挂载卷之后无法读取到配置文件信息  权限也给了

**Comment by waiguoren001** [comment]
![image](https://user-images.githubusercontent.com/96926582/198939149-e32569ed-ae5c-4280-a4a6-772f9dd1b0f1.png)
![image](https://user-images.githubusercontent.com/96926582/198939256-21631206-d9e5-4693-9611-321920ac7e48.png)

运行一下docker就停止

**Comment by luxcgo** [comment]
日志也看到了，并没有传入运行的指令。

要传入具体的命令才会运行啊。

**Reply by waiguoren001** [reply]
![image](https://user-images.githubusercontent.com/96926582/198957920-0df5afba-f3be-4ee6-a938-7397396804a1.png)
![image](https://user-images.githubusercontent.com/96926582/198958113-364d2100-d605-42b5-9b5a-00afdd9e51de.png)
我的确是设置了配置文件

**Reply by luxcgo** [reply]
有两个问题
1. 不是配置文件，指的是运行docker的命令。你可以把怎么运行的docker命令发一下。
2. 目前还没支持douyu

**Comment by seekwindow** [comment]
群晖图形介面创建的容器不能修改命令行的，必须从SSH登入后用命令行重建容器
